### DISCUSSION TITLE
[Windows 11] generate_projects.bat is not providing a list of presets

### DISCUSSION THREAD

**Discussion by soopnoods** [NVIDIA-Omniverse/PhysX#104] [Q&A] [answered]
Background:
I am trying to build physx for the first time and I am running into some issues. I am unsure if I am missing any steps as the readme is rather sparse https://github.com/NVIDIA-Omniverse/PhysX/blob/release/104.2/physx/documentation/platformreadme/windows/README_WINDOWS.md. 

Question:
generate_project.bat is not providing a list of possible presets, how can I fix this?
![image](https://user-images.githubusercontent.com/35354621/222006749-c9783b3c-b6b1-4707-aecd-7590bd57a46d.png)

### Library and Version

PhysX v5.1.0

### Operating System

Windows 11
Python 3.10.9
Developer Command Prompt for VS 2022

`cmake --version`
cmake version 3.22.1

`nvcc --version`
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Thu_Nov_18_09:52:33_Pacific_Standard_Time_2021
Cuda compilation tools, release 11.5, V11.5

### Steps to Trigger Behavior

 1. Clone repo using github desktop
 2. conda activate env
 4. .\generate_projects.bat

### Expected Behavior

some list of preset

### Actual Behavior

The output:
```
Preset parameter required, available presets:
Enter preset number:

```

**Comment by ayoub-belarbi** [comment]
Hello @soopnoods, 
Tried to reproduce what you faced but I'm not able to do so, everything works as expected. From what you pasted; it seems that \buildtools\cmake_generate_projects.py is unable to find the presets under "buildtools/presets/*.xml". Can you check that these presets exists? 
`glob.glob("buildtools/presets/*.xml)` will search for the directory "buildtools/presets" located in the current working directory. As far as I know `glob.glob` is a built-in Python function that is available in all Python environments, including Anaconda. However you can try to run this outside an Anaconda environment if the presets exists.
Thanks

**Reply by calvinchang33** [reply]
Hi, I'm not really sure what changed between last time and this time, but I tried just following the directions in a clean enviornment and everything worked out this time, i.e:  
1) clone repo
2) run .bat
3) select from the following:

> Preset parameter required, available presets:
> (0) vc15win64 <--- VC15 Win64 PhysX general settings
> (1) vc16win64 <--- VC16 Win64 PhysX general settings
> (2) vc17win64 <--- VC17 Win64 PhysX general settings

thanks for your time and assistance.

**Comment by ayoub-belarbi** [comment, marked as answer]
Awesome!
